test/Helper/Navigation/AbstractTestCase.php

```diff
@@ -175,7 +179,7 @@ protected function getAcl(): array
 
         $acl->addResource(new GenericResource('guest_foo'));
         $acl->addResource(new GenericResource('member_foo'), 'guest_foo');
-        $acl->addResource(new GenericResource('admin_foo', 'member_foo'));
+        $acl->addResource(new GenericResource('admin_foo'));
```

Comment:
It looks like it doesn't matter, but I think that was the original intention:
```suggestion
        $acl->addResource(new GenericResource('admin_foo'), 'member_foo');
```

Reply:
Yes it was, but the string fixtures being compared against fail when `admin_foo` is given a parent role, so I just deleted the extra argument

Reply:
I mean the difference between the brackets that leads to different arguments:

```php
$acl->addResource(new GenericResource('admin_foo', 'member_foo'));
// vs:
$acl->addResource(new GenericResource('admin_foo'), 'member_foo');
```

Reply:
See also: https://github.com/laminas/laminas-view/pull/207/files#diff-3208225af5a3ef0c924761633ad6b04cbfa5eab3d3bafcc49c79441e89257ee7L2692-L2694

Reply:
Acknowledged. I am aware of the original mistake. Removing the offending and redundant parameter affects nothing, but correcting the mistake yields different navigation structures due to the change to the ACL configuration, which in turn causes tests to fail needlessly. i.e. it's not worth correcting the misplaced parenthesis.